packages/react-core/src/components/MultipleFileUpload/MultipleFileUploadStatus.tsx

```diff
@@ -55,8 +55,8 @@ export const MultipleFileUploadStatus: React.FunctionComponent<MultipleFileUploa
 
   const toggle = (
     <div className={styles.multipleFileUploadStatusProgress}>
-      <div className={styles.multipleFileUploadStatusProgressIcon}>{icon}</div>
-      <div className={styles.multipleFileUploadStatusItemProgressText}>{statusToggleText}</div>
+      <div className="pf-v5-c-multiple-file-upload__status-progress-icon">{icon}</div>
+      <div className="pf-v5-c-multiple-file-upload__status-progress-text">{statusToggleText}</div>
```

Comment:
Changed to hardcoded classes for now as I think the styling for these classes will likely be removed for #6247, which will make them unavailable on the styles object.

Reply:
Rather than hardcoding the whole class:

```suggestion
      <div className={`${styles.multipleFileUpload}__status-progress-icon`}>{icon}</div>
      <div className={`${styles.multipleFileUpload}__status-progress-text`}>{statusToggleText}</div>
```
Should work